sdk/servicebus/Microsoft.Azure.ServiceBus/src/MessageReceivePump.cs

```diff
@@ -4,25 +4,27 @@
 namespace Microsoft.Azure.ServiceBus
 {
     using System;
+    using System.Collections.Generic;
     using System.Diagnostics;
+    using System.Linq;
     using System.Threading;
     using System.Threading.Tasks;
     using Core;
     using Primitives;
 
     sealed class MessageReceivePump
     {
-        readonly Func<Message, CancellationToken, Task> onMessageCallback;
+        readonly Func<IList<Message>, CancellationToken, Task> onMessageCallback;
```

Comment:
`IList<Message>` or `Message[]`?  Used `IList<Message>` because it is returned by `ReceiveAsync`, but could see value in `Message[]`.

Reply:
`IList<Messages>` is more aligned with the client.